Sources/PrivacyStats/internal/DailyBlockedTrackersEntity.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  DailyBlockedTrackersEntity.swift
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Common
+import CoreData
+
+@objc(DailyBlockedTrackersEntity)
+final class DailyBlockedTrackersEntity: NSManagedObject {
+
+    @nonobjc class func fetchRequest() -> NSFetchRequest<DailyBlockedTrackersEntity> {
+        return NSFetchRequest<DailyBlockedTrackersEntity>(entityName: "DailyBlockedTrackersEntity")
+    }
+
+    class func entity(in context: NSManagedObjectContext) -> NSEntityDescription {
+        return NSEntityDescription.entity(forEntityName: "DailyBlockedTrackersEntity", in: context)!
```

Comment:
```suggestion
        NSEntityDescription.entity(forEntityName: "DailyBlockedTrackersEntity", in: context)!
```

Reply:
Also, are we confident we can force unwrap here?

Reply:
Yes, we do it for all entity descriptions across BSK.